prac_07/project_management.py

```diff
@@ -77,8 +77,18 @@ def update_project():
     print("Update project")
 
 
-def add_new_project():
-    print("add new project")
+def add_new_project(incomplete_projects, completed_projects):
+    print("Let's add a new project")
+    name = input("Name: ")
+    start_date = input("Start date: ")
+    priority = int(input("Priority: "))
+    cost_estimate = float(input("Cost estimate: "))
+    percent_complete = int(input("Percent complete: "))
+    project = Project(name, start_date, priority, cost_estimate, percent_complete)
+    if project.is_complete():
+        completed_projects.append(project)
+    else:
+        incomplete_projects.append(project)
```

Comment:
Do you need to save the projects in two lists?  You may consider doing the separation once in the display function.